AccountService.js

```diff
@@ -40,4 +40,72 @@ export class AccountService {
     });
     return response.data;
   }
+
+  /**
+   * @method accounts
+   * @param {Object} config
+   * @param {string} config.url the base url
+   * @param {string} config.email the user's email
+   * @param {string} config.after an account's DID
+   * @param {number} config.limit how many accounts to return
+   * @return {Array} data
+   * @description calls on the adminr route and returns all
+   * accounts that match the email passed in.
+   * the cursor should be an account's DID
+   * and the limit is number.
+   */
+  async accounts({url = this.config.urls.base, email, after, limit}) {
+    const response = await axios.get(url, {
+      params: {email, after, limit},
+      headers: {'Accept': 'application/ld+json, application/json'}
+    });
+    return response.data;
+  }
+
+  /**
+   * @method update
+   * @param {Object} config
+   * @param {string} config.url
+   * @param {string} config.id
+   * @param {number} config.sequence
+   * @param {Array<Object>} config.patch
+   * @return {Void}
+   * @description id is an account's DID,
+   * sequence is an integer
+   * patch is an array of json-patches
```

Comment:
Combine with the @param definition?  Mainly we want to avoid having to change things in several different places when there is a change.

Reply:
will do